docs-source/content/userguide/cicd/mutate-the-domain-layer.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Mutate the domain layer"
+date: 2019-04-11T13:43:41-04:00
+draft: false
+weight: 4
+description: "How to mutate the domain layer."
+---
+
+If you need to mutate the domain layer, and keep the same domain encryption keys,
+then there are some choices about how to implement that, as alluded to previously.
+Let's explore those in some more detail now.
+
+The first option is to implement each mutation as a delta to the previous state.
```

Comment:
It's not clear to me how to implement each mutation as a delta to the previous state. It will be helpful if you provide concrete simple samples.

Reply:
wait/defer